apps/pie-microsite/src/_11ty/shortcodes/tokensTable/index.js

```diff
@@ -123,25 +116,27 @@ const buildTokenPill = tokenScssName => `<span class="c-tokensTable-token">${tok
  * @param {string} config.tokenScssName - the design token SCSS name i.e. '$color-black'
  * @param {string} config.tokenDisplayName - the display name of the token i.e. 'Black'
  * @param {object} config.tokenMetadata - the metadata for the token. data such as descriptions
+ * @param {object} config.path - path to the category i.e.  'path:color.alias.default' / 'path:color.alias.dark'
  * @returns {string} - the list item HTML string
  */
 const buildTokenListElements = ({
     token,
     tokenType,
     tokenScssName,
     tokenDisplayName,
-    tokenMetadata = {}
+    tokenMetadata = {},
+    path = {}
 }) => {
     const tokenPill = buildTokenPill(tokenScssName);
-    const tokenExampleElement = buildTokenExampleElement(token, tokenType, tokenMetadata);
+    const tokenExampleElement = buildTokenExampleElement(token, tokenType, tokenMetadata, path);
 
     // TODO - description is just an example of how we might use the metadata
     // We would likely wanted to move them into a colour specific handler similar to how we build
     // the colour token example. Please consider them placeholder for now.
-    const tokenDescription = buildTokenDescriptionElement(tokenType, tokenMetadata);
+    const tokenDescription = buildTokenDescriptionElement(tokenMetadata);
 
     return deindentHTML(`
-    <li class="c-tokensTable-row c-tokensTable-item">
+    <li class="c-tokensTable-row c-tokensTable-item" style="${getExampleColumnSize(tokenType)}">
```

Comment:
for readability I'd probably have stored the function call in a variable and placed that variable in here rather than the function call directly